ext/standard/string.c

```diff
@@ -1866,6 +1866,44 @@ PHP_FUNCTION(str_contains)
 }
 /* }}} */
 
+/* {{{ proto boolean str_starts_with(string haystack, string needle)
+   Checks if haystack strats with needle */
```

Comment:
Typo "strats"
Also, to be consistent with the existing surrounding functions, maybe rather "Checks if _a string_ starts with _another_"? (and similarly for str_ends_with below)